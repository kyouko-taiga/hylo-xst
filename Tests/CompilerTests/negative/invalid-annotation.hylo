//! no-std

@  blah
fun f() {}

struct A {
  @blah let x: Void
  @blah typealias T = Void
}

trait P {
  @blah type X
}

@blah extension Void {}

@blah given Void is P {}
