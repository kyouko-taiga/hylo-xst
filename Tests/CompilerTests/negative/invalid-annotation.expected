negative/invalid-annotation.hylo:3.2-4: error: extraneous whitespace between '@' and annotation identifier
@  blah
 ~~
negative/invalid-annotation.hylo:7.3-8: error: invalid annotation
  @blah let x: Void
  ~~~~~
negative/invalid-annotation.hylo:8.3-8: error: invalid annotation
  @blah typealias T = Void
  ~~~~~
negative/invalid-annotation.hylo:12.3-8: error: invalid annotation
  @blah type X
  ~~~~~
negative/invalid-annotation.hylo:15.1-6: error: invalid annotation
@blah extension Void {}
~~~~~
negative/invalid-annotation.hylo:17.1-6: error: invalid annotation
@blah given Void is P {}
~~~~~
